lib/Common/Localization.hs

```diff
@@ -74,6 +74,10 @@ _ERROR_SERVICE_USER__REQUIRED_ADMIN_ROLE_OR_HASH_IN_QUERY_PARAMS =
 
 _ERROR_SERVICE_USER__REQUIRED_HASH_IN_QUERY_PARAMS = "A hash query param has to be provided"
 
+-- DataManagementPlan
+_ERROR_TRANSFORMATION_FAILED = "Couldn't transform to desired document format"
+_ERROR_UKNOWN_FORMAT = "Unprocessable DMP format"
```

Comment:
Please keep the prefix same as it is in other error messages. Here it would be `_ERROR_SERVICE_DMP__UKNOWN_FORMAT`

Reply:
Fixed in https://github.com/DataStewardshipWizard/dsw-server/pull/6/commits/860e458918a6ed80969eb340eff7a90e880b2922